plexapi/audio.py

```diff
@@ -120,7 +120,7 @@ def sync(self, bitrate, client=None, clientId=None, limit=None, title=None):
 
         section = self._server.library.sectionByID(self.librarySectionID)
 
-        sync_item.location = 'library://%s/item/%s' % (section.uuid, quote_plus(self.key))
+        sync_item.location = f'library://{section.uuid}/item/{quote_plus(self.key)}'
```

Comment:
Isnt the curlybrace at the wrong place? Shouldnt it be quoute_plus({self.key})

Reply:
F strings should not be used in logging for performance reasons.

Reply:
> Isnt the curlybrace at the wrong place? Shouldnt it be quoute_plus({self.key})

No, the brackets are correct. It has to evaluate `quote_plus` before formatting.

> F strings should not be used in logging for performance reasons.

The performance difference (faster or slower) is negligible.

https://github.com/PyCQA/pylint/issues/2354#issuecomment-414526879